plugins/vault-plugin-auth-mock/go.mod

```diff
@@ -3,7 +3,10 @@ module github.com/hashicorp/vault-guides/plugins/vault-plugin-auth-mock
 go 1.12
 
 require (
-	github.com/hashicorp/go-hclog v0.8.0
-	github.com/hashicorp/vault/api v1.0.2
-	github.com/hashicorp/vault/sdk v0.1.11
+	github.com/go-ldap/ldap v3.0.2+incompatible // indirect
+	github.com/hashicorp/go-hclog v0.14.1
+	github.com/hashicorp/vault/api v1.0.5-0.20200519221902-385fac77e20f
+	github.com/hashicorp/vault/sdk v0.1.14-0.20210325185647-d3758c9bd369
+	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
+	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
```

Comment:
Can you do a `go mod tidy` to see if it cleans up some of these indirect deps?

Reply:
Done 👍